### PR TITLE
DI 22 adding debug  removing retries from solid config - for now

### PR DESCRIPTION
**Pull request by bahill**
## Why

[Relevant ticket - DI-22](https://broadworkbench.atlassian.net/browse/DI-22)

## Checklist
- [x] Documentation has been updated as needed.

### CODE REVIEW

**Review by aherbst-broad — APPROVED** [review]

File: orchestration/hca_manage/validation.py [hunk 1]
```diff
@@ -4,31 +4,37 @@
...
+        ignore_inputs: bool,
+        client: Client,
+    ) -> Any:
         """
         Run the validation pre-checks on the staging area
         :param path: Google Cloud Storage path for staging area
         """
         adapter = StagingAreaValidator(
             staging_area=path,
+            # total_retries=retries,
```
Comment (aherbst-broad) [inline orchestration/hca_manage/validation.py:24]: Can this commented out parameter be removed?

File: orchestration/hca_orchestration/solids/validate_ingress.py [hunk 2]
```diff
@@ -1,39 +1,51 @@
...
 
 
 @solid(
-    required_resource_keys={'staging_area_validator', 'gcs'},
-    config_schema={
-        "staging_area": String
-    }
+    required_resource_keys={"staging_area_validator", "gcs"},
+    config_schema={"staging_area": String},
+    # config_schema={"staging_area": String, "total_retries": int},
```
Comment (aherbst-broad) [inline orchestration/hca_orchestration/solids/validate_ingress.py:15]: Same question about the commented out code.

**Review by aherbst-broad — APPROVED** [review]
If the commented out code isn't needed, I suggest removing it.

File: orchestration/hca_manage/validation.py (hunk 1 above)
Comment (bahill) [reply orchestration/hca_manage/validation.py:24]: I'm leaving it for now as I'd like to revisit this feature on my next go around with this codebase

**Pull request merged**